website/App.js

```diff
@@ -3,23 +3,60 @@ import * as React from 'react'
 
 const context = require.context('./examples', false, /\.js$/)
 const examples = context.keys().map(k => {
-  const title = startCase(k.replace(/\.js$/, ''))
+  const url = k.replace(/\.js$/, '').slice(2)
```

Comment:
Why the slice?

Reply:
To remove the `./` from the key, for example `./bars` => `bars`

Reply:
Ah, I didn't realize that `startCase` was stripping that. Nice, thanks.